browser/init.js

```diff
@@ -1,7 +1,6 @@
 var jQuery = require("funcunit/browser/jquery");
 
 var FuncUnit = window.FuncUnit || {};
```

Comment:
I'm not sure I follow.  global.js requires funcunit (`require("funcunit");`) which does this:
```
window.FuncUnit = window.S = window.F = FuncUnit;
```
..which is what I assume you mean by "FuncUnit's global".


..it does seem like it should export something, though, so I can do this:
```
require("funcunit");

var FuncUnit = window.FuncUnit || {};
window.jQuery = jQuery;

module.exports = FuncUnit;
```

?